ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -656,9 +675,9 @@ open class NavigationBar: UINavigationBar, TwoLineTitleViewDelegate {
             if isLeftItem {
                 let isRTL = effectiveUserInterfaceLayoutDirection == .rightToLeft
                 button.contentEdgeInsets = UIEdgeInsets(top: 0,
-                                                        left: isRTL ? 0 : Constants.leftBarButtonItemLeadingMargin,
+                                                        left: isRTL ? Constants.leftBarButtonItemTrailingMargin : Constants.leftBarButtonItemLeadingMargin,
                                                         bottom: 0,
-                                                        right: isRTL ? Constants.leftBarButtonItemLeadingMargin : 0)
+                                                        right: isRTL ? Constants.leftBarButtonItemLeadingMargin : Constants.leftBarButtonItemTrailingMargin)
```

Comment:
Could you factor the `isRTL` logic out? Maybe something like 
```
let leftInset: CGFloat
let rightInset: CGFloat
if effectiveUserInterfaceLayoutDirection == .rightToLeft {
  leftInset = Constants.leftBarButtonItemTrailingMargin
  rightInset = Constants.leftBarButtonLeadingMargin
} else {
  leftInset = Constants.leftBarButtonItemLeadingMargin
  rightInset = Constants.leftBarButtonTrailingMargin
}
button.contentEdgeInsets = UIEdgeInsets(top: 0, left: leftInset, bottom: 0, right: rightInset)
```

Reply:
At this point we might as well factor out the `isLeftItem` check too. It actually saves on binary size. 😁